test-specifications/token-service/customFee.md

```diff
@@ -0,0 +1,57 @@
+| denominatingTokenId | string | optional          | The ID of the token to use to assess the fee. The value "0.0.0" SHALL be used to specify the token being created if used in a `TokenCreateTransaction`. Hbar SHALL be used if no value is provided. |
+
+### Fractional Fee Object Definition
+
+| Parameter Name   | Type   | Required/Optional | Description/Notes                                                                                                                                                                                                                                                                                                                     |
+|------------------|--------|-------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| numerator        | int64  | required          | The numerator of the fraction of the amount to be assessed as a fee.                                                                                                                                                                                                                                                                  |
+| denominator      | int64  | required          | The denominator of the fraction of the amount to be assessed as a fee.                                                                                                                                                                                                                                                                |
+| minimumAmount    | int64  | required          | The minimum amount to assess as a fee.                                                                                                                                                                                                                                                                                                |
+| maximumAmount    | int64  | required          | The maximum amount to assess as a fee. 0 implies no maximum.                                                                                                                                                                                                                                                                          |
+| assessmentMethod | string | required          | How should the fee be assessed? It MUST be one of `inclusive` or `exclusive`. `inclusive` means the fee is included in the amount specified by `numerator` and `denominator`, while `exclusive` means the sender will be charged an additional amount of tokens in addition to the amount specified by `numerator` and `denominator`. |
```

Comment:
`inclusive` means the fee is subtracted from the amount of tokens transferred.
No transfer of tokens is included in numerator or denominator; the fraction of those two is multiplied by the transfer amount to calculate the fee.

Note: I'm assuming `inclusive` `==>` `net_of_transfers` = false and `exclusive` `==>` `net_of_transfers` = true

###Example
// 4.5% fractional fee, calculated in "barigo" with a 5 "barigo" minimum and 50000 "barigo" maximum
numerator == 45
denominator == 1000
minimum_amount = 500000000
maximum_amount = 5000000000000

Transfer 400,000 "barigo" tokens, custom fee = 18,000 "barigo" (400,000 * 45 / 1000)

assessmentMethod = inclusive (default)
sender charged 400,000 "barigo"; recipient receives 382,000 "barigo". 

assessmentMethod = exclusive
sender charged 418,000 "barigo"; recipient receives 400,000 "barigo"

Reply:
Updated the language here to hopefully be a bit clearer